src/Components/SingleArticle.jsx

```diff
@@ -1,18 +1,26 @@
 import { useParams } from "react-router-dom";
-import { getArticlesByID } from "../utls/api";
+import { getArticlesByID, getCommentsByID } from "../utls/api";
 import { useState, useEffect } from "react";
 import Container from "react-bootstrap/Container";
+import CommentCard from "./CommentCard";
 
 const SingleArticle = () => {
   const { article_id } = useParams();
   const [searchArticle, setArticle] = useState({});
+  const [comments, setComments] = useState([]);
 
   useEffect(() => {
     getArticlesByID(article_id).then((article) => {
       setArticle(article);
     });
   }, [article_id]);
 
+  useEffect(() => {
+    getCommentsByID(article_id).then((iDComments) => {
+      setComments(iDComments);
```

Comment:
This useEffect can be combined with the one above it, the API call functions can be made one after the other